client_app/src/components/volunteer/hooks/useShelterData.js

```diff
@@ -17,18 +18,18 @@ export const useShelterData = (defaultRadius) => {
 
   const fetchData = () => {
     setLoading(true);
-    const newEndpoint = `${GETHELP_API}v2/facilities?page=0&pageSize=1000&latitude=${latitude}&longitude=${longitude}&radius=${radius}`;
+    const newEndpoint = `${SERVER}/shelter`;
```

Comment:
This is the right change to get the data from our server, instead of the GetHelp API.